spec/models/course_spec.rb

```diff
@@ -38,94 +38,71 @@
   end
 
   before(:all) do
-    @course = FactoryGirl.build(:course, department: "EECS", course_number: 132)
-    @course.course_instances = [FactoryGirl.build(:course_instance, end_date: Date.today - 1),
-                                FactoryGirl.build(:course_instance, end_date: Date.today + 365)]
-    # Testing unhappy paths
-    @course_bad = FactoryGirl.build(:course, department: 132, course_number: "EECS")
+    @course = Course.create! department: 'EECS', course_number: 132
+  end
+
+  describe '#first_professor' do
+    it 'returns the first real one' do
+      allow(@course).to receive(:professors).and_return(double(order_by_realness: [double(name: 'Real'),
+                                                                                  double(name: 'Name'),
```

Comment:
Line is too long. [103/100]<br>Align the elements of an array literal if they span more than one line.